### PR TITLE
Append single IP addresses with /32

### PR DESCRIPTION
**Pull request by C-gyorfi**

### CODE REVIEW

**Review by yusufsheiqh — APPROVED** [review]

File: app/models/client.rb [hunk 1]
```diff
@@ -22,5 +24,11 @@ def validate_ip
     end
   end
 
+  def append_ip_range
+    return if ip_range.nil?
+
+    self.ip_range = "#{ip_range}/32" unless ip_range.match?(/^.*\/[0-9]{1,2}$/)
```
Comment (emileswarts) [inline app/models/client.rb:30]: Think we can leverage the IPAddress gem here to sidestep regexes:

ip = IPAddress::IPv4.new(ip_range)
self.ip_range = "#{ip.to_s}/#{ip.prefix}"

File: app/views/clients/_form.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div class="govuk-form-group <%= field_error(f.object, :ip_range) %>">
     <%= f.label :ip_range, "IP / Subnet CIDR", class: "govuk-label" %>
     <div id="ip-range-hint" class="govuk-hint">
-      Must be an IPv4 address with a subnet; for a single IP address, use /32. For example, '127.0.0.1/32'.
+      Must be an IPv4 address
```
Comment (emileswarts) [inline app/views/clients/_form.html.erb:5]: Think we should leave the example text saying both are accepted:
127.0.0.1 OR 127.0.0.1/16

File: app/models/client.rb (hunk 1 above)
Comment (emileswarts) [reply app/models/client.rb:30]: Also does this method run after validations? 
Feel like this method is doing both validation and appending

Comment (yusufsheiqh) [reply app/models/client.rb:30]: This happens after validation but before save, so this checks if the given `ip_range` contains a subnet/suffix, and if not, then it appends the `/32`.

**Pull request merged**